mis_builder/models/mis_report_instance.py

```diff
@@ -877,7 +878,35 @@ def _compute_matrix(self):
     def compute(self):
         self.ensure_one()
         kpi_matrix = self._compute_matrix()
-        return kpi_matrix.as_dict()
+        ret = kpi_matrix.as_dict()
+
+        ret["notes"] = self.get_notes()
+        return ret
+
+    def get_notes(self) -> dict:
+        self.ensure_one()
+        if not self.env.user.has_group("mis_builder.group_read_annotation"):
+            return {}
+
+        annotations = self.env["mis.report.instance.annotation"].search(
+            [
+                ("period_id", "in", self.period_ids.ids),
+            ]
+        )
+        annotation_context = self._get_annotation_context()
+        annotations = annotations.filtered(
+            lambda rec: rec.annotation_context == annotation_context
+        )
+
+        return {
+            KpiMatrix._make_cell_id(
+                annotation.kpi_id.id,
+                False,
+                annotation.period_id.id,
+                annotation.subkpi_id and annotation.subkpi_id.id,
+            ): {"note_text": annotation.note, "note_sequence": sequence}
```

Comment:
```suggestion
            ): {"text": annotation.note, "sequence": sequence}
```

In the contexts where we use that we know we are talking about notes, so this would like make things lighter.